includes/admin_settings.php

```diff
@@ -332,7 +347,56 @@ function tsml_settings_page()
                         </form>
                     </div>
 
-                    <div class="postbox stack">
+                    <div class="postbox stack">                        
+                        <!-- Entity Form -->
+                        <?php
+                        $tsml_entity = tsml_get_entity();
+                        ?>
+                        <div class="stack compact">
+                            <h2>
+                                <?php _e('Service Entity Information', '12-step-meeting-list') ?>
+                            </h2>
+                            <p>
+                                <?php echo sprintf(__('Enter information for your service entity here to help identity the meeting source when sharing feeds with others.', '12-step-meeting-list'), get_post_type_archive_link('tsml_meeting')) ?>
+                            </p>
+
+                            <form method="post" class="stack compact" action="<?php echo $_SERVER['REQUEST_URI'] ?>">
+                                <?php wp_nonce_field($tsml_nonce, 'tsml_nonce', false) ?>
+                                <h3>
```

Comment:
could these `h3`s be `label`s instead? the smaller type should look good as well

i would keep it `Entity Name` but change:

`Entity Contact Email` -> `Administrative Contact Email`
`Entity Contact Phone` -> `Public Phone Number`
`Entity Location` -> `Your Service Area`
`Entity Website` -> `Website Address`

Reply:
"Your" felt redundant on looking at it, like it could be in all labels. This OK, or you think the "your" belongs?
![image](https://github.com/user-attachments/assets/7cb69766-a1c1-4b6b-8d8d-d2471debda77)

Reply:
that's good! maybe make them `label`s though, if they're not already